Tensile/Common.py

```diff
@@ -493,6 +493,30 @@ def getArchitectureName(gfxName):
     # Can always be True, set to False for debugging or comparison
     "OptPreLoopVmcnt":            [False, True],
 
+    # For MatrixInstruction and SIA3, number of GlobalReadInstruction between mfma
+    # the purpose of this parameter is to control density of global read instruction scheduling
+    # Scheduling global read back to back can have better memory efficiency
+    # However, when full of vmem FIFO, it will block other instruction to be issued
+    # Range from 0.01 to 32
+    #         0.1 means 1 GR per 10 mfma
+    #           5 means 5 GR per 1 mfma
+    "GlobalReadPerMfma":       [ i/100 for i in range(1,3200)],
+    #
+    # For MatrixInstruction and SIA3, number of LocalWriteInstruction between mfma
+    # the purpose of this parameter is to control density of local write instruction scheduling
+    # In PGR1, we want to schedule local write more denser, so we can have more
+    #          latency to hide global read
+    # In PGR2, since LW is followed by GR, every LW has same whole loop latecy
+    #          to hide global read. We want to schedule LW less denser, can
+    #          avoid full of vmem FIFO.
+    # Range from 0.01 to 32
+    #         0.1 means 1 LW per 10 mfma
+    #           5 means 5 LW per 1 mfma
+    # -1 will derived an optimized value internally
+    # -2 is used to support old logic yaml
```

Comment:
please check if we really need to keep this 
if no, please remove -2 and let -2 config for will derived an optimized value and override LWPM silently.

Reply:
done